src/ebay-icon/types.ts

```diff
@@ -681,8 +649,12 @@ export type Icon =
     'unselectAll24' |
     'upload16' |
     'upload24' |
-    'verifiedCondition16Fit' |
-    'verifiedCondition24Fit' |
+    'venmo12Colored' |
```

Comment:
question outside of this - now that this has most of payment method icons do we still need our sprite images?

Reply:
Probably not, me and @montoya332 were discussing that maybe our payment sprites can use skin instead now